x-pack/filebeat/module/okta/system/ingest/pipeline.yml

```diff
@@ -4,6 +4,503 @@ processors:
   - set:
       field: event.ingested
       value: "{{_ingest.timestamp}}"
+  - script:
+      # drops null/empty values recursively
```

Comment:
Comments like this make good `description` fields. If you look at the pipeline in the editor that's part of Kibana they get rendered. 

On that note here's also a tag field that's pretty helpful when a failure occurs and you have multiple processors of that type. Or if you are looking at metrics for the pipeline.